ui/pages/token-allowance/token-allowance.js

```diff
@@ -128,17 +145,36 @@ export default function TokenAllowance({
       fullTxData.originalApprovalAmount = dappProposedTokenAmount;
     }
 
+    if (customTokenAmount) {
+      txData.customTokenAmount = customTokenAmount;
```

Comment:
Should this be `fullTxData` instead of `txData`?

Updating `txData` here does not seem to have any effect

Reply:
Good catch!
Yes, I accidentally used `txData` instead of `fullTxData`.